.github/workflows/validation.yml

```diff
@@ -0,0 +1,62 @@
+name: Validation
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+env:
+  BUILD_TYPE: Debug
+  FRAMEWORK_PATH: /__w/restG4/restG4/framework
+
+defaults:
+  run:
+    shell: bash -ieo pipefail {0}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: lobis/root-geant4-garfieldpp:cxx14_ROOTv6-25-01_Geant4v10.4.3
+
+    strategy:
+      matrix:
+        framework-branch: [master, development]
+
+    steps:
+      - name: Source ROOT and Geant4
+        run: |
+          source /root/.bashrc
```

Comment:
There is no better way to do it?

Reply:
Not that I found, seems like declaring the environment variables is not enough (or I am not declaring all of them). So far this works, but if you can think of a better way let me know.

Reply:
I don't have much experience on this topic, but I can have a look...